src/Search/Manager.php

```diff
@@ -112,7 +112,17 @@ public function callback($name, $config = [])
      */
     public function custom($name, $config = [])
     {
-        $this->_config[$name] = new $config['className']($name, $config, $this);
-        return $this;
+        if (isset($config['className'])) {
+            $this->_config[$name] = new $config['className']($name, $config, $this);
+               return $this;
```

Comment:
I'll fix that, happened probably because of some search and replace for tabs. God, I hate PSR2 so much...